tests/components/twitch/test_sensor.py

```diff
@@ -41,6 +42,9 @@ async def test_streaming(
     assert sensor_state.attributes["entity_picture"] == "stream-medium.png"
     assert sensor_state.attributes["game"] == "Good game"
     assert sensor_state.attributes["title"] == "Title"
+    assert sensor_state.attributes["started_at"] == datetime(
```

Comment:
Same comment as previously, I would have expected a iso formatted string here

Reply:
I had written it that way to match the existing assertion later in the file for `following_since`. 

https://github.com/home-assistant/core/pull/121265/commits/e48a50c266ab35b4a581803e3bb9ea4a9fb006a5#diff-b1ac5ab4e5ed3195f189b8d6a33d5b336fbc4de941bc5e2989ef1dbbd6bcc90fR93

Is there a style guide for dates/times I should be referring to?

Reply:
Aah check that makes it consistent I guess 👍 

../Frenck